client/codegen_pinvoke/Main.cs

```diff
@@ -112,10 +117,8 @@ public void Setup(CppSharp.Driver driver)
             options.GenerateClassTemplates = false;
             options.GenerateFinalizers = false;
 
-            // hard coding "unity_states" as a stopgap until we decide which target
-            // provides all the symbols for the necessary libraries
-            var module = options.AddModule("unity_states");
-            module.Libraries.Add($"libunity_states{Codegen.DYLIB_EXTENSION}");
+            var module = options.AddModule(this.single_lib_name);
+            module.Libraries.Add($"lib{this.single_lib_name}{Codegen.DYLIB_EXTENSION}");

```

Comment:
See #137, we should not specify the extension any more. 

```suggestion
            module.Libraries.Add($"lib{this.single_lib_name}");
```